### DISCUSSION TITLE
Next.js ServerSideRendering app na AppService

### DISCUSSION THREAD

**Discussion by szympulka** [AzureCommunityPL/group#235] [Pomoc]
Hej, borykam się z deployem aplikacji napisanej w next.js na AppService. MS ma w dokumentacji informację jak zrobić deploy, ale tylko dla [statycznych stron](https://docs.microsoft.com/en-us/azure/static-web-apps/deploy-nextjs). Czy ktoś z Was robił coś podobnego i może nakierować? To podejście niestety również, się nie sprawdziło [https://parveensingh.com/next-js-deployment-on-azure-app-service/](https://parveensingh.com/next-js-deployment-on-azure-app-service/)

**Comment by kamil-mrzyglod** [comment]
Ale jaki masz dokładnie problem bo z postu niewiele wynika?

**Reply by szympulka** [reply]
Faktycznie, nie sprecyzowałem. AppService nie łyka mi tych plików i dostaje HTTP 500 - This page isn’t working right now.

**Reply by kamil-mrzyglod** [reply]
500 na App Service może być spowodowane wieloma czynnikami typu zły runtime / niewłaściwy deployment / błąd w startupie aplikacji. Dobrze by było dobrać się do logów aplikacji aby zobaczyć gdzie dokładnie jest problem. Możesz to spróbować zrobić nawet z poziomu **Logs** w Azure Portal. Bez tego nadal będzie ciężko to zdiagnozować.

**Reply by szympulka** [reply]
Podepnę i sprawdzę
